gapis/api/cmd_id.go

```diff
@@ -53,5 +53,7 @@ func (id CmdID) String() string {
 // TODO(apbodnar) find a more appropriate place for this
 type CommandSubmissionKey struct {
 	SubmissionOrder uint64
-	CommandBuffer   int64
+	CommandBuffer   uint64
+	RenderPass      uint64
+	Framebuffer     uint64
```

Comment:
This is probably the wrong place for this (For all of them in practice).
It leaks Vulkan-isms out of the Vulkan package.

Renderpasses/submission indices / renderpasses/etc are all handled currently in vulkan.go when generating the sync-data. I think we can probably get this to work well with that data, rather than re-implementing it outside

Reply:
If you can suggest an alternative place for this type, I'd appreciate it.  Given the scope of profiling_data.go, importing the vulkan package causes a circular dependency.